app/globals.css

```
@@ -72,3 +73,17 @@ a {
 ::-webkit-scrollbar-thumb:hover {
   background: #555;
 }
+
+:root {
+  --background: white;
+  color-scheme: light dark;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)